modules/ROOT/examples/live-demos/local-upload/index.js

```diff
@@ -7,11 +7,9 @@ tinymce.init({
   images_upload_url: 'postAcceptor.php',
 
   /* we override default upload handler to simulate successful upload*/
-  images_upload_handler: (blobInfo, success, failure) => {
-    setTimeout(() => {
+  images_upload_handler: (blobInfo) => new Promise((resolve) => {
       /* no matter what you upload, we will turn it into TinyMCE logo :)*/
-      success('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
-    }, 2000);
-  },
+      setTimeout(() => resolve('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png'), 2000);
```

Comment:
This still isn't a correct conversion yet sorry Ben... it should be this:
```
  images_upload_handler: (blobInfo) => new Promise((resolve) => {
      setTimeout(() => {
        /* no matter what you upload, we will turn it into TinyMCE logo :)*/
        resolve('http://moxiecode.cachefly.net/tinymce/v9/images/logo.png')
      }, 2000);
  }),
```

Reply:
No problem. It seems more readable

Reply:
Changes look god now thanks!